src/core/shaders/graphs/metallicroughness.frag.json

```diff
@@ -1924,17 +1926,6 @@
                     ],
                     "substitution": "vec4 $outputColor = kuesa_metalRoughFunction($baseColor, $metalness, $roughness, $ambientOcclusion, $worldPosition, $worldView, $worldNormal);"
                 },
-                {
```

Comment:
why are we loosing these? would that not break ES2 PBR support?

Reply:
done